powerhome/playbook
# [PLAY-313] Rails Stack - Rails 7 (No React)

#### Screens

![Screen Shot 2022-09-15 at 3 00 22 PM](https://user-images.githubusercontent.com/8194056/190476873-14f59c22-2b7c-43ce-852a-e630e2f3fee6.png)

#### Breaking Changes

No breaking changes, just implementing Playbook in a Rails 7 only project.

#### Runway Ticket URL

[[PLAY-313]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-313)

#### How to test this

- Check for `acceptance criteria` in runway ticket
- Clone this repo and run a local server

#### How to run it locally

- `bundle`
- `bin/rails server`

#### Checklist:
